profile/README.md

```diff
@@ -1,24 +1,25 @@
 # Cosmos
 
-The Cosmos org is collection of teams working on decentralized technology.
+A collection of teams working on decentralized technology for the internet of blockchains - The Interchain.
 
-## Quick Start
+| [Developer Portal](https://tutorials.cosmos.network): Quick Start | [Interchain Discord](https://discord.com/invite/interchain): Support & Community | [GitHub Discussions](https://github.com/orgs/cosmos/discussions): Technical Hub |
```

Comment:
Adding GH Discussions to the top line here as the "Technical Hub" - should resolve the prior convo, what do we think team?

Reply:
👍🏾